Formula/cddlib.rb

```diff
@@ -14,9 +14,13 @@ class Cddlib < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "0cbfdb16a069a1c098e379da5b3f12c461f639f09f36addc65e5f07f27e1f1e9"
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
```

Comment:
We should probably document why we need to regenerate `configure` here in a comment.